src/main/java/exceptions/InvalidNoteOpenException.java

```diff
@@ -0,0 +1,7 @@
+package exceptions;
+
+public class InvalidNoteOpenException extends InvalidNoteException {
+    public InvalidNoteOpenException(String message) {
+        super(message);
+    }
+}
```

Comment:
Can pull the upstream and do rebase or something cause I think some stuffs are changed? Like now the exceptions are combined to 1 in upstream.